enableEpochs.toml

```diff
@@ -285,25 +285,25 @@
     FixGasRemainingForSaveKeyValueBuiltinFunctionEnableEpoch = 1265
 
     # CurrentRandomnessOnSortingEnableEpoch represents the epoch when the current randomness on sorting is enabled
-    CurrentRandomnessOnSortingEnableEpoch = 999991
+    CurrentRandomnessOnSortingEnableEpoch = 1390
```

Comment:
This is Monday, May 20-th. We should avoid activation epochs on Mondays, public holidays and weekends, as stated in the release procedures, point 4.

Reply:
updated